release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3016,6 +3016,50 @@ This section will continue to be updated over time to provide notes on enhanceme
+[id="ocp-4-18-10-enhancements_{context}"]
+==== Enhancements
+
+*  In the bootstrap phase of the installation process, the transport layer security (TLS) between the `metal3` `httpd` server and the node's Baseboard Management Controller (BMC) is enabled by default in {product-title} 4.18 and later. The `httpd` server is on port 6183 instead of port 6180 when TLS is enabled. Disable the TLS setting by adding 'disableVirtualMediaTLS: true' to the Provisioning custom resource (CR) file that is created on the disk. (link:https://issues.redhat.com/browse/OCPBUGS-39404[OCPBUGS-39404])
+
+
+[id="ocp-4-18-10-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the Prometheus remote-write proxy configuration was not correctly applied to the Prometheus user workload custom resource (CR), which caused communication and data collection problems in the cluster. With this release, the user workload monitoring (UWM) Prometheus configurations, including user workload Prometheus, correctly inherits the proxy settings from the cluster proxy resource. (link:https://issues.redhat.com/browse/OCPBUGS-38655[OCPBUGS-38655])
+
+* Previously, when running Red Hat Enterprise Linux CoreOS (RHCOS) in an active environment, the `rpm-ostree-fix-shadow-mode.service` that used to run caused the `rpm-ostree-fix-shadow-mode.service` to fail. With this release, the `rpm-ostree-fix-shadow-mode.service` does not activate when RHCOS does not run from an installed environment. (link:https://issues.redhat.com/browse/OCPBUGS-41625[OCPBUGS-41625])
+
+* Previously, an incorrect component import in `SimpleSelect.tsx` caused an undefined function `r` in `react-dom.production.min.js`. This component caused error messages on the *Dashboards* and *Metrics* pages related to dropdown lists. With this release, the dropdown lists on the affected pages function correctly, eliminating the error message. (link:https://issues.redhat.com/browse/OCPBUGS-42845[OCPBUGS-42845])
+
+* Previously, an error in the image pull secret controller's secret token rotation logic caused a temporary, invalid token for authentication, which caused disruptions in the image pull process. With this release, the updated image pull secret controller guarantees a smooth and continuous image pull process, because it eliminates the period when the token is not valid while the token rotates. (link:https://issues.redhat.com/browse/OCPBUGS-54304[OCPBUGS-54304])
+
+* Previously, an error occurred in {hcp-capital}-managed clusters because of the omission of `shutdown-watch-termination-grace-period` in the `kube-apiserver` configuration. This led to unstable shutdown of applications in {hcp}-managed clusters. With this release, an update improves the shutdown process of applications in {hcp}-managed clusters, providing a grace period for the `kube-apiserver` configuration. During a shutdown, the application stability is improved and potential errors are decreased. (link:https://issues.redhat.com/browse/OCPBUGS-53404[OCPBUGS-53404])
```

Comment:
```suggestion
* Previously, an error occurred in {hcp}-managed clusters because of the omission of the `shutdown-watch-termination-grace-period` setting in the `kube-apiserver` configuration. This error led to the unstable shutdown of applications in {hcp}-managed clusters. With this release, an update improves the shutdown process of applications in {hcp}-managed clusters, providing a grace period for the `kube-apiserver` configuration. During a shutdown, the application stability is improved and potential errors are decreased. (link:https://issues.redhat.com/browse/OCPBUGS-53404[OCPBUGS-53404])
```